- Revised item list_item_attractionplaces

nyeinnyeinaungwin123456 opened the pull request:
-Add Search Icon on FAB,
-Adapter
-Fragment
-HomeActivity

nyeinnyeinaungwin123456 committed:
- Revised item list_item_attractionplaces
-Add Search Icon on FAB,
-Adapter
-Fragment
-HomeActivity

vincent-paing commented on PADC_FP_Travelling/app/src/main/java/com/padc/travelling/activities/HomeActivity.java:
```diff
@@ -160,31 +159,33 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
 
     @Override
     public void onTapAttractionPlaces(AttractionPlacesVO attractionPlacesVO, int position) {
+        Intent intent = new Intent(TravellingApp.getContext(), AttractionDetailActivity.class);
```
Should use static-factory methods (i.e AttractionDetailActivity.newIntent())

nyeinnyeinaungwin123456 committed:
- Add Tour package implementation
-add value folder
-revised HomeActivity

nyeinnyeinaungwin123456 committed:
-implement tour package detail activity with view pager
-implement search implementation with fab
-implement call when button is pressed
-revised AttractionDetailActivity with static factory method
-Although I try TourPackagePagerDetailActivity with static factory method, I cannot call textview with putExtra in static factory method. So I use normal Intent.
-If you have some suggestion, please show me the way.
